### PR TITLE
Limit height of user page section on user profiles

### LINKED ISSUE #2815: Userpage (me! section) should be collapsible
As evidenced by [vaxei](https://osu.ppy.sh/users/vaxei)'s profile.

(also collapses by default, like old website)

### PR DESCRIPTION
**Pull request by nekodex**
...'cuz some people like to play silly buggers

fixes #2815

### CODE REVIEW

File: resources/assets/less/bem/page-extra.less [hunk 1]
```diff
@@ -80,6 +85,12 @@
     width: 100%;
   }
 
+  &__content-overflow-wrapper {
+    max-height: 400px;
```
Comment (shortthu) [inline resources/assets/less/bem/page-extra.less]: Why use 400px since the max height of _most_ old userpage is 192px?

Comment (Xinnoh) [reply resources/assets/less/bem/page-extra.less]: From what I see, the userpage doesn't have the option to expand and is locked at 400.
400 is approx the size of an expanded userpage on old web

File: resources/assets/less/bem/page-extra.less [hunk 2]
```diff
@@ -35,6 +35,11 @@
     padding: 0;
   }
 
+  &--me-section {
+    padding-right: 0px;
```
Comment (nanaya) [inline resources/assets/less/bem/page-extra.less]: hmmm

Comment (nanaya) [reply resources/assets/less/bem/page-extra.less]: I think most of scrollbars like does follow the padding (as in doesn't stick to the edge).

File: resources/assets/less/bem/page-extra.less [hunk 3]
```diff
@@ -97,5 +119,9 @@
     right: 0;
     top: 0;
     cursor: move;
+
+    .@{top}--userpage & {
+      margin-right: 30px;
```
Comment (nanaya) [inline resources/assets/less/bem/page-extra.less]: Apparently the margin is different between mobile and desktop. Also both have their own variables.

**Pull request merged**